mauiservice.ini

```diff
@@ -0,0 +1,7 @@
+[kirjastonhoitaja]
```

Comment:
This has nothing to do with dockerizing the project and also refers to files outside of the project. Should this be included?